pkg/api/pod/util.go

```diff
@@ -26,6 +26,37 @@ import (
 	"k8s.io/kubernetes/pkg/security/apparmor"
 )
 
+// ContainerVisitor is called with each container spec, and returns true
```

Comment:
We probably should be using an api specific package for this, not the internal package.  Or is this the internal types utility and I just missed the versioned ones?

Reply:
Nm, I just missed the versioned ones.